config/windows.json

```
@@ -16,6 +16,8 @@
     { "wmClass": "gnome-terminal-server", "wmTitle": "Preferences – General", "mode": "float" },
     { "wmClass": "gnome-terminal-preferences", "mode": "float" },
     { "wmClass": "Guake", "mode": "float" },
-    { "wmClass": "zoom", "mode": "float" }
+    { "wmClass": "zoom", "mode": "float" },
+    { "wmClass": "firefox", "wmTitle": "About Mozilla Firefox", "mode": "float" },
+    { "wmClass": "firefox", "wmTitle": "Picture-in-Picture", "mode": "float" }
```

Using the new *not* filter #306 

```suggestion
    { "wmClass": "firefox", "wmTitle": "!Mozilla Firefox", "mode": "float" }
```